services/thermal_mgr/thermal_mgr.c

```diff
@@ -41,21 +43,77 @@ void initThermalSystemManager(lm75bd_config_t *config) {
 
 }
 
+/*
+ * @brief Sends event to the thermal manager queue. Fails is queue is full.
+ *
+ * @param event - Pointer to a thermal_mgr_event_t to add to the queue.
+ */
 error_code_t thermalMgrSendEvent(thermal_mgr_event_t *event) {
-  /* Send an event to the thermal manager queue */
-
-  return ERR_CODE_SUCCESS;
+    /* Send an event to the thermal manager queue */  
+    if( event == NULL ) {
+        return ERR_CODE_INVALID_ARG;
+    }
+
```

Comment:
I would check to make sure that the queue has already been created (i.e. the queue handle is not null)